.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+## Overview
+
+_A brief description of the pull request. Try to keep it non-technical. Please mark your pull request with the appropriate label(s)_
+
+## Before
+
+_The current status. Please provide screenshots or gifs ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)) where appropriate_
+
+## After
+
+_What will be changed_
```

Comment:
I think it's better to repeat the screenshot/gif bit, we don't want people to think that they should provide them only for the "Before" section